package.json

```diff
@@ -3,11 +3,13 @@
   "dependencies": {
     "@koa/cors": "^3.0.0",
     "@koa/router": "^8.0.2",
+    "http-link-header": "^1.0.2",
     "koa": "^2.11.0",
     "koa-logger": "^3.2.1",
-    "rdf-namespaces": "^1.5.0"
+    "rdf-namespaces": "^1.6.0"
```

Comment:
You might also be interested in [`@zazuko/rdf-vocabularies`](https://www.npmjs.com/package/@zazuko/rdf-vocabularies) as an alternative.

Reply:
Will take a look separately. (PRs always welcome of course. 😃)